test/verify_extra_test.c

```diff
@@ -182,9 +182,12 @@ static int test_self_signed(const char *filename, int use_trusted, int expected)
 {
     X509 *cert = load_cert_from_file(filename); /* may result in NULL */
     STACK_OF(X509) *trusted = sk_X509_new_null();
-    X509_STORE_CTX *ctx = X509_STORE_CTX_new();
+    X509_STORE_CTX *ctx = NULL;
     int ret;
 
+    if ((ctx = X509_STORE_CTX_new()) == NULL)
```

Comment:
Please use TEST_ptr() and return 0.. This is a common pattern in the test files.